docs/users_guide/introduction/perturbation.md

```diff
@@ -162,20 +162,78 @@ with the `_adj` suffix:
 - If adjusted parameters are not present in the surface file, eCLM falls
   back to the original parameters or pedotransfer functions
 
-#### Parameter Priority and Fallback Option
-
-The code in `SoilStateInitTimeConstMod.F90` follows this priority order:
-
-1. **First priority**: Read `_adj` parameters (if present) → applies
-  to all `nlevgrnd` layers → **overwrites** organic matter mixing
-  results
-2. **Second priority**: Read original parameters → applies to first 10
-  layers (`nlevsoifl`) → undergoes organic matter mixing
-3. **Fallback**: Use pedotransfer functions if parameters are absent
-  from the file
+#### Namelist Configuration
+
+The soil hydraulic parameter reading behavior is controlled by two
+namelist settings in the `clm_soilstate_inparm` section of the `lnd_in`
+namelist file:
+
+##### `parameters_in_file`
+
+**Type:** logical
+**Default:** `.false.`
+**Description:** Controls whether to read baseline hydraulic parameters
+from the surface dataset file.
+
+When set to `.true.`:
+- eCLM reads `THETAS`, `SHAPE_PARAM`, `PSIS_SAT`, and `KSAT` from the
+  surface file
+- Parameters apply to the first 10 soil layers (`nlevsoifl=10`)
+- Parameters undergo organic matter mixing
+- If any required variable is missing, the model aborts with an error
+  message
+
+When set to `.false.` (default):
+- Hydraulic parameters are computed via pedotransfer functions from
+  sand and clay fractions
+- No parameters are read from the surface file
+
+##### `parameters_in_file_adj`
```

Comment:
Are the adjusted parameters exclusively used for eCLM-PDAF runs? If it is, I think it makes more sense to use `#ifdef USE_PDAF` for its corresponding logic. Also, if `parameters_in_file_adj` are required for eCLM-PDAF runs, then there is no need to add a namelist parameter for it.

Reply:
The separated docs are much clearer--thanks for this!